docs/development/localnet-with-kubernetes.md

```diff
@@ -0,0 +1,69 @@
+# LocalNet on Kubernetes
+
+We are developing our own Kubernetes operator to manage v1 workloads both internally and for the community to make it easier to deploy pocket v1. While the operator is still in development, we can already utilize it to run our local networks locally on Kubernetes. This guide will show you how to do that.
+
+- [LocalNet on Kubernetes](#localnet-on-kubernetes)
+    - [Dependencies](#dependencies)
+    - [Running the localnet](#running-the-localnet)
+    - [Stopping / cleaning up the resources](#stopping--cleaning-up-the-resources)
+    - [Interaction with the localnet](#interaction-with-the-localnet)
+    - [How does it work?](#how-does-it-work)
+    - [Troubleshooting](#troubleshooting)
+    - [How to change configuration files](#how-to-change-configuration-files)
+
+### Dependencies
+
+* [tilt](https://docs.tilt.dev/install.html)
+* Kubernetes cluster ([different options available](https://docs.tilt.dev/choosing_clusters.html)), please make sure to run 1.23 version or older, until [this issue is resolved](https://github.com/zalando/postgres-operator/issues/2098) - otherwise Postgres DB is not going to be provisioned.
+  * `kubectl` is also required to be installed and configured to access the cluster, but that should happen automatically when you install kubernetes cluster locally.
+  * `helm` is required to template the yaml manifests for the observability stack. Here are the instructions on how to install it: https://helm.sh/docs/intro/install/.
```

Comment:
Let me know what you think of this style of writing for development docs. Personally, I don't think people like to read - just `copy-pasta` as fast as possible

```suggestion
  * `helm` - required to template the yaml manifests for the observability stack.
  	* Installation instructions: : https://helm.sh/docs/intro/install/.
```